cmd/shared.go

```diff
@@ -91,3 +91,24 @@ func fileExists(filename string) bool {
 func stripNewLines(stripString string) string {
 	return strings.TrimSuffix(stripString, "\n")
 }
+
+func nullStrCheck(s string) *string {
+	if s == "" {
+		return nil
+	}
+	return &s
+}
+
+func nullUintCheck(i uint) *uint {
+	if i == 0 {
+		return nil
+	}
+	return &i
+}
+
+func requiredInputCheck(field string, value string) {
+	if value == "" || value == "0" {
+		fmt.Println(fmt.Sprintf("Missing argument: %s is not defined", field))
+		os.Exit(1)
+	}
+}
```

Comment:
Change this to return an error and fmt.Errorf the error out.

All the commands should use `RunE` which returns an error with the command usage output

Reply:
Updated now